frame/clearing-house/src/types.rs

```diff
@@ -69,12 +70,18 @@ pub struct Market<T: Config> {
 	/// Minimum amount of quote asset to exchange when opening a position. Also serves to round
 	/// a trade if it results in closing an existing position
 	pub minimum_trade_size: T::Decimal,
-	/// Net position, in base asset, of all traders. Used to compute parameter adjustment costs and
-	/// funding payments from/to the Clearing House
-	pub net_base_asset_amount: T::Decimal,
-	/// The latest cumulative funding rate of this
-	/// market. Must be updated periodically.
-	pub cum_funding_rate: T::Decimal,
+	/// Total position, in base asset, of all traders that are long. Must be positive. Used to
+	/// compute parameter adjustment costs and funding payments from/to the Clearing House
+	pub base_asset_amount_long: T::Decimal,
+	/// Total position, in base asset, of all traders that are short. Must be negative. Used to
+	/// compute parameter adjustment costs and funding payments from/to the Clearing House
+	pub base_asset_amount_short: T::Decimal,
+	/// The latest cumulative funding rate for long positions in this market. Must be updated
+	/// periodically
+	pub cum_funding_rate_long: T::Decimal,
+	/// The latest cumulative funding rate for short positions in this market. Must be updated
+	/// periodically
+	pub cum_funding_rate_short: T::Decimal,
 	/// Amount, in quote asset, of fees collected from trades
```

Comment:
Nitpick but worth mentioning: 

Citing the [Rust Style Guide](https://doc.rust-lang.org/1.0.0/style/style/comments.html#sentence-structure) regarding comments: 

> All doc comments, including the summary line, should begin with a capital letter and end with a period, question mark, or exclamation point.

I think it might be good to add a ending period in all comments here.

Reply:
Seems there are a lot of periods missing :grimacing: